apisix/plugins/ext-plugin/init.lua

```diff
@@ -65,6 +65,18 @@ local type = type
 
 
 local events_list
+local exclude_resp_header = {
+    ["connection"] = true,
+    ["content-length"] = true,
+    ["transfer-encoding"] = true,
+    ["location"] = true,
+    ["server"] = true,
+    ["www-authenticate"] = true,
+    ["content-encoding"] = true,
+    ["content-type"] = true,
+    ["content-location"] = true,
+    ["content-language"] = true,
```

Comment:
Should we also exclude headers like: "Set-Cookie"?

Reply:
Maybe some users want to use the external plugin to modify the Cookie, but it's also unsafe.

Reply:
Yes,I want use the external plugin to modify the Cookie and there is a better way?

Reply:
Maybe we can allow using  "Set-Cookie"? Setting Cookie won't break the upstream's response.

Reply:
I think we can just let it go and see the reaction of community.